src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -13,6 +13,8 @@
 public class PersonCard extends UiPart<Region> {
 
     private static final String FXML = "PersonListCard.fxml";
+    private static final String[] tagColors = { "red" , "orange", "yellow", "green", "blue", "violet", "indigo",
+                                                "purple" , "pink" , "grey"};
```

Comment:
Just a suggestion: try to choose the tag colors which users can see the word inside clearly :)